deploy-image/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk
+
+RUN apt-get update
+
+RUN  apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+RUN  curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
+
+RUN git clone git://github.com/yyuu/pyenv.git .pyenv
```

Comment:
```suggestion
```

Reply:
I think we should consider using this rather than `pyenv-installer`, which seems to clone other repos that aren't necessary - https://github.com/pyenv/pyenv-installer/blob/master/bin/pyenv-installer#L54-L59.

Reply:
Yeah I agree - in this case it's the `RUN  curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash` that needs to be removed @berroar